### PR TITLE
add bane

### PR DESCRIPTION
**Pull request by Moshe-Immerman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,6 +238,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 ### Security
 
 * [Aqua Security 💲](https://www.aquasec.com) - Securing container-based applications from Dev to Production on any platform
+* [bane](https://github.com/jessfraz/bane) - Custom & better AppArmor profile generator for Docker container by [@jessfraz][jessfraz]
```
Comment (devhkr) [inline README.md]: :nerd_face: symbol is missing for CLI tools.

```
[bane](https://github.com/jessfraz/bane) :nerd_face: - Custom
```

Comment (veggiemonk) [inline README.md]: What does `better` mean ? Please be specific. Better than what and how ?
Also, a description in the PR would be nice 😄

**Comment by Moshe-Immerman** [comment]
The alternative is to write an apparmor profile by hand as far I as I can see

**Comment by veggiemonk** [comment]
still missing the :nerd_face:

**Comment by devhkr** [comment]
LGTM

**Pull request merged**